homeassistant/components/backup/manager.py

```diff
@@ -236,22 +241,34 @@ def __init__(self, hass: HomeAssistant) -> None:
         """Initialize the backup manager."""
         super().__init__(hass=hass)
         self.backup_dir = Path(hass.config.path("backups"))
-        self.loaded_backups = False
+        self.temp_backup_dir = Path(hass.config.path("tmp_backups"))
 
     async def async_upload_backup(self, *, slug: str, **kwargs: Any) -> None:
```

Comment:
I think this method should be removed, there's no need for a special method which uploads to all agents

Reply:
This is confirmed to meant as a way for supervisor to trigger agent upload when a supervisor backup is completed. We don't need that now, but it's also not relevant for this PR.